apparmor.d/groups/lxqt/lxqt-config

```diff
@@ -0,0 +1,67 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/lxqt-config
+profile lxqt-config @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/bus-session>
+  include <abstractions/bus-accessibility>
+  include <abstractions/graphics>
+  include <abstractions/lxqt>
+  include <abstractions/nameservice-strict>
+
+  @{exec_path} mr,
+
+  owner @{user_config_dirs}/qt6ct/qt6ct.conf    rw,
+
+  @{bin}/lxqt-admin-user rPx,
+  @{bin}/ibus-setup rPx,
+  @{bin}/lxqt-config-monitor rPx,
+  @{bin}/pcmanfm-qt rPx,
+  @{bin}/lxqt-admin-time rPx,
+  @{bin}/lxqt-config-input rPx,
+  @{bin}/lxqt-config-locale rPx,
+  @{bin}/lxqt-config-brightness rPx,
+  @{bin}/lxqt-config-session rPx,
+  @{bin}/lxqt-config-file-associations rPx,
+  @{bin}/lxqt-config-powermanagement rPx,
+  @{bin}/lxqt-config-appearance rPx,
+  @{bin}/lxqt-config-globalkeyshortcuts rPx,
+  @{bin}/lxqt-config-notificationd rPx,
+  @{bin}/obconf-qt rPx,
+  @{bin}/nm-connection-editor rPx,
+  @{bin}/pavucontrol rPx,
+  @{bin}/pavucontrol-qt rPx,
+  @{bin}/system-config-printer rPx,
+  @{bin}/nm-connection-editor rPx,
+  @{bin}/ControlPanel rPx,
+  @{bin}/qt6ct rix,
+  @{bin}/xdg-open rPx,
```

Comment:
This needs to be handled as:
```sh
  @{open_path}                                   rPx -> child-open-any,
```
See: https://apparmor.pujol.io/development/internal/#open-resources